pallets/basic-sto/src/lib.rs

```diff
@@ -0,0 +1,178 @@
+// Copyright (c) 2020 Polymath
+
+//! # STO Module
+
+#![cfg_attr(not(feature = "std"), no_std)]
+#![recursion_limit = "256"]
+
+use codec::{Decode, Encode};
+use frame_support::{
+    decl_error, decl_event, decl_module, decl_storage, dispatch::DispatchResult, ensure,
+};
+use frame_system::{self as system, ensure_signed};
+use pallet_identity as identity;
+use pallet_settlement::{Leg, SettlementType};
+use polymesh_common_utilities::{
+    traits::{asset::Trait as AssetTrait, identity::Trait as IdentityTrait, CommonTrait},
+    Context,
+};
+use polymesh_primitives::{IdentityId, Ticker};
+use sp_runtime::traits::{CheckedMul, Saturating};
+use sp_std::prelude::*;
+
+type Identity<T> = identity::Module<T>;
+type Settlement<T> = pallet_settlement::Module<T>;
+
+pub trait Trait:
+    frame_system::Trait
+    + CommonTrait
+    + IdentityTrait
+    + pallet_timestamp::Trait
```

Comment:
I don't think you need timestamp module as it is already present in the IdentityTrait

Reply:
In fact, I don't need timestamp module at all. I ended up not using it for the POC module. Will remove.